main/wallet-api.md

```diff
@@ -1,30 +1,389 @@
+either via an autodeposit or manually approved.
+
+### `getDepositFacet(brandBoardId)`
+- `brandBoardId` `{String}`
+- Returns: `{Promise<string>}`
+
+Returns the board ID for the deposit facet of a purse in the user's Wallet that accepts payments
+of the brand specified by the `brandBoardId` parameter. **tyg todo: What if
+the Wallet has more than one purse for the specified by board ID brand? Or is
+the parameter really the id for a purse that takes that brand?**
+
+### `getIssuers()`
+- Returns: `{Array<[Petname, Issuer]>}`
+
+Returns an array of all the Issuers and their petnames associated with this Wallet.
+
+### `getIssuer(petname)`
+- `petname` `{Petname}`
+- Returns: `{Issuer}`
+
+Returns the issuer with the specified petname associated with this Wallet.
+
+### `getPurses()`
+- Returns: `{Array<[Petname, Purse]>}`
+
+Returns all the purses associated with this wallet.
+
+### `getPurse(pursePetname)`
+- `pursePetName`  `{String}`
+- Returns `{Purse}`
+- Errors: Throws an error if there is no purse with the given petname.
+
+Returns the `purse` object with the given petname
+    
+## WalletBridge API commands    
+    
+These methods can be used by an untrusted Dapp without breaching the wallet's 
+integrity.  They are also exposed via the iframe/WebSocket bridge that a 
+Dapp UI can use to access a Wallet.
+ 
+### `addOffer(offer)`
+- `offer` `{OfferState}`
+- Returns: `{Promise<string>}`
+
+Adds an offer to the Wallet, returning **tyg todo: Not sure what the returned string is for**
+
+### `addOfferInvitation(offer, invitation)`
+- `offer` `{OfferState}`
+- `invitation` `{ERef<Payment>}`
+- Returns: `{Promise<string>}`
+
+Add the specified invitation to the specified offer. **tyg todo: Not sure what the returned string is for**
+
+### `getDepositFacetId(brandBoardId`
+- `brandBoardId` `{string}`
+- Returns: `{Promise<string>}`
+
+Returns the Board ID to use to receive payments of the specified by its Board ID brand.
+
+### `getPursesNotifier()`
+- Returns: `{Promise<Notifier<Array<PursesFullState>>>}`
+
+Returns a notifier that follows changes the purses in the Wallet.
+
+### `getOffersNotifier()`
+- Returns: `{Promise<Notifier<array<OfferState>>>}`
+
+Returns a notifier that follows changes to the offers received by the Wallet.
+
+### `suggestIssuer(petname, issuerBoardId)`
+- `petname` `{Petname}`
+- `issuerBoardId` `{string}`
+- Returns: `void`
+
+Introduce an ERTP issuer with a suggested petname to the Wallet.
+
+### `suggestInstallation(petname, installationBoardID)`
+- `petname` `{Petname}`
+- `installationBoardId` `{string}`
+- Returns: `void`
+
+Introduce a Zoe contract installation with a suggested petname to the Wallet.
+
+### `suggestInstance(petname, instanceBoardId)
+- `petname` `{Petname}`
+- `instanceBoardId` `{string}`
+- Returns: `void`
+
+Introduce a Zoe contract instance with a suggested petname to the Wallet.
+ 
+
+**tyg todo: old API stuff starts here. Should any of it be retained?**
```

Comment:
No, none should be retained.

Please move the rest to a separate, unlinked document in case we want to revive any of it when we make the APIs public again.